### DISCUSSION TITLE
WarmUps for specific rank or permission

### DISCUSSION THREAD

**Discussion by mkko120** [Zrips/CMI#5429] [Enhancement]
Is there a possibility to add a function for warmup for a specific rank or with specific permission? I've seen that it's already done in cooldowns, so why not add it in WarmUps?

-/\-\/-/\-\/-/\-\/-/\-\/-/\-\/-/\-\/-
kindest regards
mkko

**Comment by dkalaxdk** [comment]
A way of doing it now.
Would be to add the cooldown to the commands globally, and then add bypass permissions to the players you do not want to have it.

**Reply by mkko120** [reply]
But the way I wanna do it is to give e.g. VIP rank 1s cooldown and Player rank 5s cooldown
